components/table/table-col-sort-button.js

```diff
@@ -28,6 +30,14 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				reflect: true,
 				type: Boolean
 			},
+			/**
+			 * The type of data in the column.
+			 *  @type {'words'|'numbers'|'dates'}
+			 */
+			columnDataType: {
+				reflect: true,
+				type: String
+			},
```

Comment:
This could probably just be `dataType` since the component is already column sort.
Reminder to add the `attribute`.